projects/itoa/Dockerfile

```diff
@@ -0,0 +1,5 @@
+FROM rustlang/rust:nightly
```

Comment:
This needs to inherit from our builder gcr.io/oss-fuzz-base/base-builder-rust ([example](https://github.com/google/oss-fuzz/blob/master/projects/rustls/Dockerfile#L17))